examples/keras_to_tensorflow.patch

```diff
@@ -0,0 +1,22 @@
+--- tensorflow-yolo-v3/tools/model_converter/keras_to_tensorflow.py	2021-08-17 04:17:22.186256673 +0530
++++ keras_to_tensorflow_modified.py	2021-08-17 04:14:43.686249508 +0530
+@@ -22,7 +22,9 @@
+ 
+ sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..'))
+ from common.utils import get_custom_objects
+-
++# tf.enable_eager_execution()
```

Comment:
If this line is commented out in the patch, it doesn't have to be a part of the patch in the first place, right?